behandlingsprosess/src/main/java/no/nav/ung/sak/domene/behandling/steg/varselrevurdering/VarselRevurderingStegImpl.java

```diff
@@ -76,4 +106,85 @@ private List<UngdomsprogramBekreftetPeriodeEndring> finnBekreftelser(Behandling
             .flatMap(it -> it.getBekreftetPeriodeEndringer().stream()).toList();
     }
 
+    private void opprettTaskForEtterlysning(BehandlingskontrollKontekst kontekst, LocalDateTimeline<Boolean> endretUngdomsprogramTidslinje) {
+        UngdomsprogramPeriodeGrunnlag ungdomsprogramPeriodeGrunnlag = ungdomsprogramPeriodeRepository.hentGrunnlag(kontekst.getBehandlingId()).orElseThrow();
+
+        List<Trigger> triggere = prosessTriggereRepository.hentGrunnlag(kontekst.getBehandlingId())
+            .stream()
+            .map(ProsessTriggere::getTriggere)
+            .flatMap(Collection::stream)
+            .toList();
+
+        LocalDateTimeline<Set<Etterlysning>> etterlysningerLocalDateTimeline = triggere
+            .stream()
+            .map(trigger -> {
+                    Etterlysning etterlysning = mapTilEtterlysning(kontekst, trigger, ungdomsprogramPeriodeGrunnlag);
+                    if (etterlysning == null) {
```

Comment:
Eg liker bedre å filtrere ut dei vi er interessert i i streamen i staden for å returnere null